content/blog/2016/2016-01-10-beautiful-jenkins-dashboard.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "A beautiful Jenkins dashboard"
+author: kj187
+---
+
+
+Jenkins offers a handful of third party dashboards, but none of them is really beautiful and flexible from my point of view.
```

Comment:
@kj187 I think that paragraph explains why I, as a developer creating a dashboard, would like to use Dashing. It doesn't really explain why I, as a viewer/user, would prefer a Dashing-based dashboard. 

For example, it's a whole lot easier for me to add additional widgets to a Dashing dashboard and come up with something interesting that *includes* Jenkins data. I cannot do that in the existing Jenkins-built-in-dashboards which are Jenkins-content specific.

Things along those lines I agree would be helpful